expansion/src/main/java/no/unit/nva/expansion/model/ExpandedMessage.java

```diff
@@ -57,6 +59,8 @@ public class ExpandedMessage implements JsonSerializable {
     private Instant modifiedDate;
     @JsonProperty(RESOURCE_TITLE)
     private String resourceTitle;
+    @JsonProperty(STATUS)
+    private MessageStatus status;
 
     @JacocoGenerated
```

Comment:
Remove jacocoGenerated annotation?

Reply:
This one is used in builder only, but is not "tested".